src/main/java/com/adapter/adapter/Persistence.java

```diff
@@ -0,0 +1,21 @@
+package com.adapter.adapter;
+
+import com.adapter.User;
+import com.adapter.dynamoDB.IDynamoDB;
+import com.adapter.mysql.IMysql;
+
+public class Persistence implements IPersistence{
+    private IDynamoDB dynamoDB;
+    private IMysql mysql;
+
+    public Persistence(IDynamoDB dynamoDB, IMysql mysql) {
+        this.dynamoDB = dynamoDB;
+        this.mysql = mysql;
+    }
+
+    @Override
+    public void persistenceInformation(User user) {
+        System.out.println(mysql.save(user.getUserInformation()));
+        System.out.println(dynamoDB.save(user));
```

Comment:
I'd spect here a new dto being transmitted to `dynamoDB`, that dto should be created from `user` class in order to maintain consistency, by sending the whole dto dynamo now should handle the whole dto, which is not ideal.